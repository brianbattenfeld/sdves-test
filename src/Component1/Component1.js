import React from "react";
import "./Component1.css";

const component1 = (props) => {
  return (
    <div className="Component1">
      <p>This is a sample component</p>
    </div>
  );
};

export default component1;
