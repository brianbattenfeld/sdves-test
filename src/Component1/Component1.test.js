import React from "react";
import { render } from "@testing-library/react";
import renderer from "react-test-renderer";
import { mount, shallow } from "enzyme";
import Component1 from "./Component1";

it("renders Component1 test", () => {
  const { getByText } = render(<Component1 />);
  const componentElement = getByText(/sample component/i);
  expect(componentElement).toBeInTheDocument();
});

it("renders correctly, the same as last time", () => {
  const tree = renderer.create(<Component1 />).toJSON();
  expect(tree).toMatchSnapshot();
});
