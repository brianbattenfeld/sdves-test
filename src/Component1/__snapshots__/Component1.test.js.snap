// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly, the same as last time 1`] = `
<div
  className="Component1"
>
  <h1>
    There are 
     items for the sample component.
  </h1>
</div>
`;
